Makefile

```diff
@@ -0,0 +1,8 @@
+EXAMPLE = core/core_input_gamepad
```

Comment:
It seems this file shouldn't be in the PR

Reply:
Yea, not sure how it got added. I'll fix that.